resources/scripts/add_unreal_integration.py

```diff
@@ -0,0 +1,353 @@
+# This file is provided by Epic Games, Inc. and is subject to the license
+# file included in this repository.
+
+import argparse
+import logging
+import os
+import re
+import shutil
+
+"""
+Add the Unreal integration to a SG TK default2 config.
```

Comment:
100% agree with Josh

Reply:
Hello, I share your concerns and this script is very fragile since it relies on a certain config structure, with no guarantee it will not change.
Not sure if the TK team would be happy to take ownership for it. Which in some way would be similar to adding tk-unreal to the default2 config (possibly disabled).
That being said, I think it is right to have it in the engine repo (if we decide to keep it): this is the place where users would look for documentation on how to enable the integration.
So not sure what should be done here...

Reply:
I feel pretty strongly it shouldn't live here. Perhaps a separate repo or gist that is referenced in this repo's `README` would be a good compromise?

Reply:
No strong opinion on my side. But if it needs to be moved somewhere else, the new place will have to be created for me: I don't have write access in this github org.

Reply:
My recommendation would be a PR on `tk-config-default2`. Any reason why it's not included in there anyway? Either they accept it, or the PR can serve as reference for anyone who wants to merge. That'll make it so nobody has to own the code long term.